omnigibson/utils/usd_utils.py

```diff
@@ -124,7 +124,7 @@ def get_camera_params(viewport):
     view_proj_mat = helpers.get_view_proj_mat(view_params)
 
     return {
-        "pose": np.array(prim_tf),
+        "pose": np.array(prim_tf).T,
```

Comment:
hmmm why the transpose?

Reply:
I think empirically I just observed that the prim_tf is a transposed homogeneous matrix wrt to what we normally expect I think